scripts/deposit.js

```diff
@@ -1,38 +1,41 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable indent */
 const SnappBase = artifacts.require("SnappBase")
 const ERC20Mintable = artifacts.require("ERC20Mintable.sol")
 
 const zero_address = "0x0000000000000000000000000000000000000000"
```

Comment:
I know I put this here, but recently realized this can be replaces with `0x0` (without quotes)